### PR TITLE
Update cl-edit.owl

### LINKED ISSUE #738: Add gamma motor neuron
https://en.wikipedia.org/wiki/Beta_motor_neuron

### PR DESCRIPTION
**Pull request by dosumis**
Fixes #738

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -21851,6 +21853,13 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0008036 "extravillous cytotr
 AnnotationAssertion(rdfs:label obo:CL_0008036 "extravillous trophoblast"@en)
 SubClassOf(obo:CL_0008036 obo:CL_0000351)
 
+# Class: obo:CL_0008037 (obo:CL_0008037)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersion:Gamma_motor_neuron&oldid=906640465") obo:IAO_0000115 obo:CL_0008037 "A lower motor neuron that innervates only intrafusal muscle fibers.")
```
Comment (shawntanzk) [inline src/ontology/cl-edit.owl:21858]: Add more information

**Review by shawntanzk — CHANGES_REQUESTED** [review]
Needs more information

**Comment by dosumis** [comment]
@shawntanzk  - can you look in to adding some more recent references on gamma neurons. I'd especially like some refs (maybe text book?) that back up the statements about function.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -21853,21 +21852,25 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0008036 "extravillous cytotr
 AnnotationAssertion(rdfs:label obo:CL_0008036 "extravillous trophoblast"@en)
 SubClassOf(obo:CL_0008036 obo:CL_0000351)
 
-# Class: obo:CL_0008037 (obo:CL_0008037)
+# Class: obo:CL_0008037 (gamma motor neuron)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersion:Gamma_motor_neuron&oldid=906640465") obo:IAO_0000115 obo:CL_0008037 "A lower motor neuron that innervates only intrafusal muscle fibers.")
-AnnotationAssertion(<http://www.w3.org/2004/02/skos/core#prefLabel> obo:CL_0008037 "gamma motor neuron"@en)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1093/acprof:oso/9780195159561.001.1") Annotation(oboInOwl:hasDbXref "WikipediaVersion:Gamma_motor_neuron&oldid=906640465") obo:IAO_0000115 obo:CL_0008037 "A lower motor neuron that innervates only intrafusal muscle fibers. Unlike the alpha motor neurons, gamma motor neurons do not directly adjust the lengthening or shortening of muscles but function in adjusting the sensitivity of muscle spindles and in keeping muscle spindles taut, thereby allowing the continued firing of alpha neurons.")
```
Comment (shawntanzk) [inline src/ontology/cl-edit.owl:21857]: Is this how you add a DOI? the manual came with PMID, this only has a DOI or ISBN-13

**Comment by dosumis** [comment]
DOI is fine.  Format is correct.  Build fail on travis is due to us hitting data limits.  Now fixed & re-running.

**Comment by shawntanzk** [comment]
Oh was just doing a whole google on what the issue was, is it ok now? I thought it came down to my Docker not running well as I couldn't do the file sharing part, as I couldn't access it, as I couldn't access hyper-V, as I'm using windows home version (It was a lot of googling haha). If it's all ok, happy to add more stuff in. On that note, how does the cell ontology repository deal with species? - most of my readings on lower motor neurons (esp on transcriptomics) are mouse based.

**Review by shawntanzk — APPROVED** [review]

**Pull request merged**